### PR TITLE
fix(css): font-stretch has been renamed to font-width

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- related to https://github.com/mdn/content/issues/35265

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Instead of this, we should wait and properly fix this as part of https://github.com/mdn/content/issues/34347 by calling `font-stretch` a legacy alias of `font-width`.

**Comment by wbamberg** [comment]
> Instead of this, we should wait and properly fix this as part of #34347 by calling `font-stretch` a legacy alias of `font-width`.

Is it worth landing some version of this though, while we wait for browsers to implement it? Because in the meantime people are confused.

**Comment by Josh-Cena** [comment]
Hmmm, ok, that also sounds fine as an interim step.

**Comment by wbamberg** [comment]
I suppose the difficulty with doing nothing until browsers implement this is that people will get confused by the formal syntax. But the difficulty with merging something like this PR now, is that people will get confused when they try to use `font-width` and it doesn't work. Again this is connected to MDN not being able to document features that don't have browser support.

So perhaps the best thing for now is for the note to say that browsers don't yet support the new name, and remember to update it when they do? At which point we could do https://github.com/mdn/content/issues/34347#issuecomment-2208743716 ?

File: files/en-us/web/css/font-stretch/index.md [hunk 1]
```diff
@@ -216,6 +216,7 @@ td {
 
 > [!NOTE]
 > The `font-stretch` property was initially defined in CSS 2, but dropped in CSS 2.1 due to the lack of browser implementation. It was brought back in CSS 3.
+> The property [has now been renamed to `font-width`](https://drafts.csswg.org/css-fonts/#font-stretch-desc) in the specifications.
```
Comment (wbamberg) [inline files/en-us/web/css/font-stretch/index.md:219]: Suggestions:
- put this note at the top of the page, and explicitly say that `font-stretch` is now defined in the specifications as an alias for `font-width`, which is identical in behavior, but that the `font-width` name is not yet supported by any browsers.
- remove the pre-existing note from here
- also pass `font-width` to csssyntax, which should fix the formal syntax error.

File: files/en-us/web/css/font-stretch/index.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ browser-compat: css.properties.font-stretch
 
 {{CSSRef}}
 
+> [!NOTE]
+> The `font-stretch` property [has now been renamed to `font-width`](https://drafts.csswg.org/css-fonts/#font-stretch-desc) in the specifications. And the `font-stretch` name remains as an alias for the `font-width` property.
```
Comment (wbamberg) [inline files/en-us/web/css/font-stretch/index.md:11]:
```suggestion
> The `font-stretch` property [has now been renamed to `font-width`](https://drafts.csswg.org/css-fonts/#font-stretch-desc) in the specifications. The name `font-stretch` has been kept as an alias for the `font-width` property.
```

File: files/en-us/web/css/font-stretch/index.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ browser-compat: css.properties.font-stretch
...
+> The new name `font-width` has not been supported by any browser yet.
```
Comment (wbamberg) [inline files/en-us/web/css/font-stretch/index.md:12]:
```suggestion
> The new name `font-width` is not yet supported by any browsers.
```

File: files/en-us/web/css/font-stretch/index.md [hunk 4]
```diff
@@ -202,21 +206,51 @@ td {
...
+.normal {
+  font-stretch: 100%;
+}
+
+.expanded {
+  font-stretch: 200%;
+}
+```
+
+{{EmbedLiveSample("Setting font stretch percentages", "100%", 200)}}
```
Comment (wbamberg) [inline files/en-us/web/css/font-stretch/index.md:248]: Good idea, but this doesn't seem to be working in the preview. Do you want to have the font locally, as in https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-alternates#enabling_swash_glyphs, or are you supposed to load it from mdn/shared-assets?

**Review by wbamberg — CHANGES_REQUESTED** [review]

File: files/en-us/web/css/font-stretch/index.md (hunk 4 above)
Comment (OnkarRuikar) [reply files/en-us/web/css/font-stretch/index.md:248]: Yes, it's there in the shared-assets.

**Review by wbamberg — APPROVED** [review]
👍 thank you @OnkarRuikar !

**Pull request merged**